plugins/api-docs/package.json

```diff
@@ -71,8 +71,7 @@
     "graphql": "^16.0.0",
     "graphql-config": "^5.0.2",
     "graphql-ws": "^5.4.1",
-    "isomorphic-form-data": "^2.0.0",
-    "swagger-ui-react": "^5.0.0"
+    "swagger-ui-react": "^5.19.0"
```

Comment:
Why just `5.19.0` and not to `5.27.0` which has been out for 2 months now?

Reply:
Given `5.27.1` is already 2 month old let's upgrade to that. 👍

Reply:
Changes applied.